Code/GraphMol/MolDraw2D/MolDraw2DUtils.h

```diff
@@ -1,5 +1,5 @@
 //
-//  Copyright (C) 2016-2019 Greg Landrum
+//  Copyright (C) 2016-2021 Greg Landrum
```

Comment:
Did you mean 2021?  Others have been changed to 2020.

Reply:
I'm LIVING IN THE FUTURE. :-)